datasets/us/ofac/press_releases/crawler.py

```diff
@@ -0,0 +1,213 @@
+- country: Countries explicitly mentioned as residence, registration, or operation. Leave empty if not stated.
+- related_url: URLs mentioned in the article specifically associated with the entity.  
+  • If multiple URLs are present, link each one only to the entity it is associated with.  
+  • If no URL is provided for an entity, leave this field empty.  
+  • Do not invent, infer, or alter URLs.
+"""
+
+
+def source_changed(review: Review, article_element: HtmlElement) -> bool:
+    """
+    The key exists but the current source data looks different from the existing version
+    in spite of heavy normalisation.
+    """
+    seen_element = fromstring(review.source_value)
+    return html_to_text_hash(seen_element) != html_to_text_hash(article_element)
+
+
+def get_or_request_review(context, html_part, article_key, url):
+    review = get_review(context, Designees, article_key, MIN_MODEL_VERSION)
+    if review is None:
+        prompt_result = run_typed_text_prompt(
+            context, PROMPT, html_part, Designees, MAX_TOKENS
+        )
+        review = request_review(
+            context,
+            article_key,
+            html_part,
+            HTML,
+            "Press Release",
+            url,
+            prompt_result,
+            MODEL_VERSION,
+        )
+    return review
+
+
+def check_something_changed(
+    context: Context,
+    review: Review,
+    article_html: str,
+    article_element: HtmlElement,
+) -> bool:
+    """
+    Returns True if the source content has changed.
+    In that case it also reprompts to log whether the extracted data has changed.
+    """
+    if source_changed(review, article_element):
+        prompt_result = run_typed_text_prompt(
+            context, PROMPT, article_html, Designees, MAX_TOKENS
+        )
+        if model_hash(prompt_result) == model_hash(review.orig_extraction_data):
+            context.log.warning(
+                "The source content has changed but the extracted data has not",
+                url=review.source_url,
+                seen_source_value=review.source_value,
+                new_source_value=article_html,
+            )
+        else:
+            # A new extraction result looks different from the known original extraction
+            context.log.warning(
+                "The extracted data has changed",
+                url=review.source_url,
+                orig_extracted_data=review.orig_extraction_data.model_dump(),
+                prompt_result=prompt_result.model_dump(),
+            )
+        return True
+    else:
+        return False
+
+
+def crawl_item(context, item, date, url, article_name):
+    entity = context.make(item.entity_schema)
+    entity.id = context.make_id(item.name, item.country)
+    entity.add("name", item.name, origin=DEFAULT_MODEL)
+    if item.nationality:
+        # Add nationality only for 'Person' schema
+        entity.add("nationality", item.nationality, origin=DEFAULT_MODEL)
```

Comment:
The comment didn't quite match the implementation

```suggestion
    nationality_prop = "nationality"
    if item.entity_schema != "Person":
        nationality_prop = "country"
    entity.add(nationality_prop, item.nationality, origin=DEFAULT_MODEL)
```